web-services/query/src/test/java/datawave/webservice/query/logic/filtered/DateRangeFilteredQueryLogicTest.java

```diff
@@ -0,0 +1,341 @@
+package datawave.webservice.query.logic.filtered;
+
+import static org.easymock.EasyMock.anyLong;
+import static org.easymock.EasyMock.anyObject;
+import static org.easymock.EasyMock.capture;
+import static org.easymock.EasyMock.eq;
+import static org.easymock.EasyMock.expect;
+import static org.easymock.EasyMock.expectLastCall;
+import static org.easymock.EasyMock.isA;
+import static org.junit.Assert.assertEquals;
+import static org.junit.Assert.assertTrue;
+
+import java.text.ParseException;
+import java.text.SimpleDateFormat;
+import java.util.ArrayList;
+import java.util.Calendar;
+import java.util.Collections;
+import java.util.Date;
+import java.util.HashSet;
+import java.util.Iterator;
+import java.util.List;
+import java.util.Map;
+import java.util.Set;
+import java.util.UUID;
+
+import org.apache.accumulo.core.data.Key;
+import org.apache.accumulo.core.data.Value;
+import org.apache.accumulo.core.security.Authorizations;
+import org.apache.commons.collections4.iterators.TransformIterator;
+import org.easymock.Capture;
+import org.easymock.EasyMockSupport;
+import org.junit.Before;
+import org.junit.Test;
+
+import datawave.core.query.configuration.GenericQueryConfiguration;
+import datawave.core.query.logic.QueryLogic;
+import datawave.core.query.logic.QueryLogicTransformer;
+import datawave.core.query.logic.composite.CompositeQueryLogic;
+import datawave.core.query.logic.filtered.DateRangeFilteredQueryLogic;
+import datawave.core.query.logic.filtered.QueryLogicFilterByDate;
+import datawave.microservice.query.Query;
+import datawave.microservice.query.QueryImpl;
+import datawave.security.authorization.AuthorizationException;
+import datawave.webservice.query.logic.composite.CompositeQueryLogicTest;
+import datawave.webservice.result.BaseQueryResponse;
+
+public class DateRangeFilteredQueryLogicTest extends EasyMockSupport {
```

Comment:
Consider using mockito? Re: https://github.com/NationalSecurityAgency/datawave/pull/2646

Reply:
not something I want to do at this time.